lib/formatter/csv/annotation_for_csv.rb

```diff
@@ -125,7 +133,12 @@ def task_label(task_info)
         translate(task_info["question"] || task_info["instruction"])
       end
 
-      def tool_label(task_info, tool_index)
+      def tool_label(task_info, tool)
+        begin
+          tool_index = tool["tool"]
+        rescue TypeError => e
```

Comment:
Are we particularly interested in the specific `e.message` arising from duck typing in this case. If not, perhaps `raise ToolFormattingError unless Hash === tool` as the first line of the method would be more readable.